packages/frontend-2/components/project/page/InviteDialog.vue

```diff
@@ -88,9 +94,22 @@ const { userSearch, searchVariables } = useUserSearch({
     limit: 5
   }))
 })
-const { collaboratorListItems } = useTeamDialogInternals({
-  props: toRefs(props)
-})
+
+const projectId = computed(() => route.params.id as string)
```

Comment:
Not ideal to rely on route params in a non-page component, because it could also be used in different pages (and the Invite Dialog is already on 2 different pages).  Preferably we want to keep the projectId prop here